src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -417,7 +426,12 @@
                 this.donationMatchingOptions = info.donationMatchingOptions;
                 this.donationMatchingRule = info.donationMatchingRule;
                 this.donationDateRange = info.donationDateRange;
+                this.labels = info.labels;
```

Comment:
This usage breaks the pattern a bit (insofar as I understand it), and is one of the reasons we believe we need to rewrite ourselves out of this pattern. To work around it here though, I would put this logic into the markup instead of the controller since we're only relying on it for user input rather than in any functions. Then you can remove the label assignment here, and in the View module too.

Example: where you've written 'disabled="{!v.batchInfo.noMatchSelected}"', instead you could use disabled="{!v.batchInfo.donationMatchingBehavior == v.batchMetadata.labels.DoNotMatch}"

Reply:
This is great feedback, and fwiw here's my justification: 

My preference is to avoid putting logic directly into the markup except for convenience in cases where the logic is straightforward. The logic for enabling/disabling the date range field gets more complex since it depends on the selected fields in the matching model selection above, so that ends up being more like:

disabled="{!or(v.batchInfo.donationMatchingBehavior == v.batchMetadata.labels.DoNotMatch, v.batchInfo.donationMatchingRule.indexOf("donation_date__c") < 0)}"

or 

disabled="{!or(v.batchInfo.donationMatchingBehavior == v.batchMetadata.labels.DoNotMatch, v.batchInfo.noMatchOnDate)}"

The latter one is kind of a hybrid approach. 

All that said, I didn't feel SUPER great about copying the labels into the BatchInfo/BatchInfoView module, but for sort of complex reasons wasn't sure if it was 'breaking the model' per se, since it's information that we now need to accurately render the view.

Ultimately I think this comes down to another vote for re-factoring, and I'm probably going to take your suggestion of just moving the logic into the markup since it gets the job done and makes the delivery quite a bit simpler.  I'm a tiny bit concerned that behavior could possibly drift between the state of the 'real' data in the info module and what's currently in the rendered markup, but I think that's probably not a valid concern. 

How do people write short comments? Asking for a friend.